thefuck/shells/generic.py

```diff
@@ -36,18 +37,20 @@ def to_shell(self, command_script):
         return command_script
 
     def app_alias(self, alias_name):
-        return """alias {0}='eval "$(TF_ALIAS={0} PYTHONIOENCODING=utf-8 """ \
-               """thefuck "$(fc -ln -1)")"'""".format(alias_name)
+        return (
+            """alias {0}='eval "$(TF_ALIAS={0} PYTHONIOENCODING=utf-8 """
+            """thefuck "$(fc -ln -1)")"'""".format(alias_name)
+        )
 
     def instant_mode_alias(self, alias_name):
         warn("Instant mode not supported by your shell")
         return self.app_alias(alias_name)
 
     def _get_history_file_name(self):
-        return ''
+        return ""
 
     def _get_history_line(self, command_script):
-        return ''
+        return ""
```

Comment:
there does seem to be a fair few superfluous changes?

Reply:
I was advised by @scorphus to run [black](https://github.com/psf/black), which is what made these superfluous linting changes.

Reply:
OK, sure, no worries.  When Im at work, I'd try and get my team to keep changes (especially inconsequential changes) to a minimum, so it's obvious what files really need reviewing.

Reply:
I agree with @davidhart82. It's a good practice to submit only changes that are directly related to the issue/feature they address.

On one hand, Black is great because it leaves no place for discussions about how to format code – out of a sudden, you stop wasting time with everything related to code formatting. On the other hand, when it comes to a codebase that's not previously formatted by it, that generates undesirable noise, such as this very one.

So it's up to the developer to include only the relevant changes.

We can discuss whether or not we should reformat the entire codebase at once and make Black part of the development cycle on another issue.